packages/py3_pip.rb

```diff
@@ -23,6 +23,7 @@ class Py3_pip < Package
      x86_64: '729a9958cae998936dc3ca300feb7c10eb8d56f74b524913cc84cb1cf6bbe553'
   })
 
+  depends_on 'python3'
```

Comment:
This line doesn't need to be here since python3 is a core package.  I think it would be better to include `depends_on 'py3_pip'` in python3.

Reply:
Python3 doesn't depend on pip3 and pip3 doesn't even work without python3 installed. If anything it's the reverse.

Reply:
Also @uberhacker why do you want to have pip as a core package?

Reply:
I know python3 doesn't depend on py3_pip but the idea is to make sure it is installed along with python3.  As @satmandu and I have said repeatedly we would like py3_pip to be a core package because the expectation is to have it there by default.

Reply:
expectation by whom?

Reply:
Expectation of those coming from using python in every other distribution.

Also that's how python is distributed by default.

Reply:
But there's also a philosophical issue here. Chromebrew is for people who have elected to use dev mode. They want to do more than what Google wants people to do on their machines.

Taking away pip goes in the direction of taking choice away from people, and I think that goes against the spirit of Chromebrew providing people more power to do what they want.

Yes I know that disk space is often limited for ChromeOS devices, but let's find ways to be clever about that.

Reply:
Debian, Ubuntu, Fedora and Alpine ship pip in a different package. I understand the concern, however running `crew install py3_pip` should hardly be taking away a user's choice to do what they want.

Reply:
Well...if you want to make it harder for users, that IMHO is not a good philosophical choice.  My only concern is if it isn't installed in core then you better be sure it's included as a dependency in the many packages that require it.  Personally, I think it makes sense to include it in core if it's a dependency in more than a handful of packages.  The footprint of pip is very light so space shouldn't really be a concern.